terraform/example/backend.tf

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# bucket = "PROJECT_ID-tfstate"
+# prefix = "env/dev/_APP_ID"
+
+terraform {
+    backend "gcs" {
+      bucket = "andrey-cp-8-9-tfstate"
+      prefix = "env/dev/quota-monitoring-solution"
```

Comment:
hard coded for now but can be also replaces in the prep env vars step in https://github.com/gitrey/quota-monitoring-solution/pull/1/files#diff-8be5c2dc8f1ca27edb4ee7dc183d76927fdddf7a851a8bb94e2a3d07b333c453R29